maintainers/maintainer-list.nix

```
@@ -2222,6 +2222,13 @@
     githubId = 68566724;
     name = "bootstrap-prime";
   };
+  boozedog = {
+    email = "code@booze.dog";
+    github = "boozedog";
+    githubId = 1410808;
+    matrix = "@boozedog:matrix.org";
+    name = "David A. Buser (boozedog)";
```

```suggestion
    name = "David A. Buser";
```
Your pseudo is already in there, there's no need to duplicate it.

---

Fixed.